e2e-tests/IIoTPlatform-E2E-Tests/deploy/DeploymentConfiguration.cs

```diff
@@ -0,0 +1,62 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests.TestExtensions {
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Common.Exceptions;
+    using System.Threading;
+    using Microsoft.Azure.Devices;
+
+    public class DeploymentConfiguration {
+
+        public DeploymentConfiguration(IIoTPlatformTestContext context) {
+            _context = context ?? throw new ArgumentNullException(nameof(context));
```

Comment:
It seems like this class only needs an instance of `RegistryManager` and not the whole `IIoTPlatformTestContext`. Please consider changing it to only get object of `RegistryManager` type.

Reply:
Now I see that `RegistryManager` is not actually one of our classes, so I would suggest to use `RegistryHelper` instead as it is supposed to be a wrapper around `RegistryManager` with additional convenience methods.

Reply:
Addressed in #955.